tsconfig.json

```diff
@@ -19,15 +15,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "baseUrl": "."
+    "baseUrl": ".",
```

Comment:
Invalid JSON (typescript still compiles with this because of the way it loads it).

Reply:
Removed the comma 👍